EIPS/eip-6963.md

```diff
@@ -46,6 +46,7 @@ Provider Discovery Library: A library or piece of software that assists a DApp t
 Each wallet provider will be announced with the following interface `EIP6963ProviderInfo`. The values in the `EIP6963ProviderInfo` MUST be used as follows:
 
 - **`uuid`** - a globally unique of the wallet provider that MUST be (UUIDv4[^rfc4122] compliant) to uniquely distinguish different [EIP-1193](./eip-1193.md) provider sessions. The cryptographic uniqueness provided by UUIDv4[^rfc4122] guarantees that two independent `EIP6963ProviderInfo` objects can be separately identified.
+- **`rdns`** - an optional identifier that is used to optimistically identify the wallet by the Decentralized Application. If supplied by the wallet it MUST utilize an reverse DNS notation identifier that matches the domain name of the wallet provider.
```

Comment:
Should we explicitly disambiguate `rdns`?
Funnily, it's not that obvious

Possible disambiguations:
* RDN — Reverse Domain Notation
* RDN — Reverse Domain Name
* RDNS — Reverse DNS (as in Reverse Domain Name System)
* RDNS — Reverse Domain Name Scheme (as referred to in this document: https://gcc.gnu.org/legacy-ml/java/2000-04/msg00095.html)

Reply:
I tried doing that above:
https://github.com/ethereum/EIPs/pull/7256#discussion_r1251624311
the RFC for domain names actually includes a section on reverse-ordered domains, which I think makes more sense than referencing the application-manifest tradition from Java onward (which, though popularized by Apple, was never actually *standardized* as such... making it more of a convention uniting various context-specific syntaxes than a universal, IETF-style syntax!)

Reply:
I did my best to define this a bit further in the latest round of changes without getting to specific here so that we can make this more extensible/specific in the future if necessary. Marking this one as resolved.